aspnetcore/mvc/views/partial.md

```diff
@@ -73,6 +73,16 @@ Since it streams the result directly, `RenderPartialAsync` may perform better in
 > [!IMPORTANT]
 > If your views need to execute code, use a [view component](xref:mvc/views/view-components) instead of a partial view.
 
+::: moniker range=">= aspnetcore-2.1"
+
+In ASP.NET Core 2.1 and later, calling `Partial` or `RenderPartial` results in an analyzer warning. For example, usage of `Partial` yields the following warning message:
+
+> Use of IHtmlHelper.Partial may result in application deadlocks. Consider using `<partial>` Tag Helper or `IHtmlHelper.PartialAsync`.
+
+Replace calls to `@Html.Partial` with `@await Html.PartialAsync` or the Partial Tag Helper. See <xref:mvc/views/tag-helpers/builtin-th/partial-tag-helper#migrate-from-an-html-helper> for Partial Tag Helper migration information.
```

Comment:
`@Rick-Anderson` seems to lean in the direction of the following sentence structure for these "see this" sentences ...

> For more information on Partial Tag Helper migration, see \<xref:mvc/views/tag-helpers/builtin-th/partial-tag-helper#migrate-from-an-html-helper>.

Reply:
btw- Can we pull headings, too?

```
<#framework-provided-services>
```
... or does it work for `xref`-type links only?

Reply:
Looks like the style I used doesn't work properly with headings. The link works, but the link text reflects the title of the doc (rather than the heading). I'll log an issue for this, as it should work.

Reply:
It would be very nice.

It would be even crazy nicer if it could do both when the bookmark is on the title ...

```
<xref:mvc/views/tag-helpers/builtin-th/partial-tag-helper#migrate-from-an-html-helper>
```
... becomes ...

```html
<a href="LINK">Partial Tag Helper in ASP.NET Core: Migrate from an html helper</a>
```
The only problem being those anchors that Rick loves (*yuck!* :smile: lol).

Reply:
@guardrex Issue https://github.com/dotnet/docfx/issues/2657 is tracking this limitation.